File: Java/Array-1/commonEnd/CommonEnd.java
package commonEnd;

public class CommonEnd {
    //Retorne true se os começarem ou terminarem com os mesmos elementos, ambos devem ter um "length" maior ou igual a 1.

    public boolean commonEnd(int[] a, int[] b) {
        return a[0] == b[0] || a[a.length-1] == b[b.length-1];
    }
}

File: Python/List-1/common_end.py
"""
Dada 2 listas numéricas ("a" e "b"), retorne true se elas começarem com o mesmo
elemento ou terminarem com o mesmo. Ambas terão tamanho de pelo menos 1.
"""


def common_end(a, b):
    return a[0] == b[0] or a[-1] == b[-1]


print(common_end([1, 2, 3], [7, 3]))
